src/commands/watch/watch.ts

```diff
@@ -28,17 +25,24 @@ export class Watch implements Command {
     const options: WatchOptions = {};
 
     this.args.forEach((arg) => {
-      if (arg.includes("--entryPoint=")) { // TODO Improve this code
-        const entryPoint = arg.split("=")[1];
-        options.entryPoint = entryPoint;
-      } else if (arg.includes("--cwd=")) {
-        const cwd = arg.split("=")[1];
-        options.cwd = cwd;
-      } else if (arg.includes("--runtimeOptions")) {
-        const runtimeOption = arg.split("=")[1].split(",");
-        options.runtimeOptions = runtimeOption;
-      } else if (arg.includes("--help")) {
+      if (/^--help/.test(arg) || /^-h/.test(arg)) {
         options.help = true;
+      } else if (/^--.+=/.test(arg)) {
+        let m = arg.match(/^--([^=]+)=([\s\S]*)$/);
+        if (m === null) return;
```

Comment:
I think regex at line 30 can be improved to check if we have value after the `=` to avoid the case where m is empty

Reply:
Changed to `/^--.+=.+/` so it will only be returned true if there is something after `=`
```ts
      } else if (/^--.+=.+/.test(arg)) {
```